### PR TITLE
Ru: update any ui .csv file

### PR DESCRIPTION
**Pull request by Sinitsaa**

### CODE REVIEW

File: translated files/UI/ui_item.csv [hunk 1]
```diff
@@ -1830,7 +1830,7 @@ WithdrawResultMessage2#0,"""アイテムを取り下げできませんでした
 Title#18,"""広告編集"""
 EditCmText#0,"""広告文を記入"""
 Decide#0,"""決定"""
-CancelText#0,"""キャンセル"""
+CCancelText#0,"""キャンセル"""
```
Comment (alama) [inline translated files/UI/ui_item.csv]: What is wrong this change in `CCancelText#0`?

Comment (Sinitsaa) [reply translated files/UI/ui_item.csv]: I don't know. These files were handed to me by our translation team.
Now I am correcting all the mistakes that they made.
Later I will explain to them what they were wrong.

File: translated files/UI/ui_item.csv [hunk 2]
```diff
@@ -1554,250 +1554,250 @@ Explanation11#1,"""<c f0df60>取り下げたいアイテムを<%Decide><c f0df60
...
+ValueRange#0,"""Выбрать вилку цен"""
+ValueMin#0,"""Мин. цена"""
+ValueMax#0,"""Макс. цена"""
+SetValueMin#0,"""Изменить"""
+SetValueMax#0,"""Изменить"""
+SetValueMinInputNumberTitle#0,"""Выбрать мин. цену"""
+SetValueMinInputNumberMessage#0,"""Введите минимальную цену"""
+SetValueMaxInputNumberTitle#0,"""Выбрать макс. цену"""
+SetValueMaxInputNumberMessage#0,"""Введите максимальную цену"""
+SearchNew#0,""Искать по времени выставления на продажу"""
```
Comment (alama) [inline translated files/UI/ui_item.csv]: `SearchNew#0` is missing a quote

File: translated files/UI/ui_item.csv [hunk 3]
```diff
@@ -1021,101 +1021,101 @@ Explanation6_v#0,"""<c f0df60>倉庫を選択後、売却したいアイテム
...
+SynchronizingMessage#0,"""Загрузка..."""
+MenuBuyItem#0,"""Купить сейчас"""
+Buy#0,"""Купить"""
+Back#1,"""Вернуться"""
+StackBuyInputNumberTitle#0,"""Купить сейчас"""
+StackBuyInputNumberMessage#0,"""Сколько предметов вы хотите купить?"""
+BuyErrorTitle#0,"""Магазин"""
+BuyErrorMessage1#0,"""Недостаточно месет."""
+BuyErrorMessage2#0,"""Нет места в инвентаре<br>либо достигнут лимит таких предметов."""
+BuyErrorMessage3#0,"""Достигнут максимум предметов \""'$(0)'""\.<br>Вы не можете хранить больше."""
```
Comment (alama) [inline translated files/UI/ui_item.csv]: `""\`????
Should this be `\""`

File: translated files/UI/ui_item.csv [hunk 4]
```diff
@@ -67,394 +67,394 @@ ITEM23#0,"""ITEM23"""
...
-BuyCheck#1,"""以下のアイテムを$(1)$(0)で購入します。<br>よろしいですか？"""
-CancelTitle#1,"""購入キャンセル確認"""
-CancelMessage#1,"""購入をキャンセルします。<br>よろしいですか？"""
-CafePointInsufficient#0,"""所持カフェポイントが不足しています。"""
+PointGuide#0,"""<c f0df60>Выберите предмет, который хотите приобрести за кафе-поинты.<c>"""
+MesetaGuide#0,"""<c f0df60>【Только при подключении через NetCafe】 Выберите предмет, который хотите приобрести.<c>"""
+MenuTitle#3,"""Главное меню"""
+MenuPointShop#0,"""Обменять кафе-поинты на предметы"""
+MenuMesetaShop#0,"""Купить предметы (Только в NetCafe)"""
+SetErrorMessage2#2,"""Выйти"""
```
Comment (alama) [inline translated files/UI/ui_item.csv]: What happens to `MenuExit#0`

**Pull request merged**